bitcoin/src/network.rs

```diff
@@ -83,6 +84,20 @@ pub enum Network {
     /// Bitcoin's regtest network.
     Regtest,
 }
+// To preserve `as usize` behavior in a composite Network enumeration,
+// The Into<usize> trait is actively implemented
+impl Into<usize> for Network {
```

Comment:
Please no. `as usize` was a horrible hack and this even exposes it.

Reply:
Okay, let me think about how to remove this behavior, because original code need to use Network as a index to match some parameters

Reply:
I'd just use `match` directly.

Reply:
Yes I agree with you, I removed the `as usize` behavior.